crates/bevy_core_pipeline/src/taa/mod.rs

```diff
@@ -38,9 +38,10 @@ use bevy_utils::tracing::warn;
 
 const TAA_SHADER_HANDLE: Handle<Shader> = Handle::weak_from_u128(656865235226276);
 
-/// Plugin for temporal anti-aliasing. Disables multisample anti-aliasing (MSAA).
+/// Plugin for temporal anti-aliasing.
 ///
-/// See [`TemporalAntiAliasSettings`] for more details.
+/// See [`TemporalAntiAliasSettings`] for more details. Any camera with this component must
```

Comment:
This is a doc on the plugin, but the docs makes it sounds like this is a component.

Reply:
moved it to the settings struct itself